build-windows.yaml

```
@@ -19,7 +19,7 @@ services:
       dockerfile: ./windows/${WINDOWS_FLAVOR}/Dockerfile
       args:
         JAVA_HOME: "C:/openjdk-17"
-        JAVA_VERSION: 17.0.10+7
+        JAVA_VERSION: 17.0.11_9
```

```suggestion
        JAVA_VERSION: 17.0.11+9
```
Should have been the result of #797 if there wasn't #798